### PR TITLE
[NDMII-2654][snmp] store devices tags on disk with mock directory

### PR DESCRIPTION
**Pull request by jedupau**

### CODE REVIEW

File: pkg/collector/corechecks/snmp/snmp_test.go [hunk 1]
```diff
@@ -56,6 +56,8 @@ func createDeps(t *testing.T) deps {
 }
 
 func Test_Run_simpleCase(t *testing.T) {
+	testDir := t.TempDir()
+	coreconfig.Datadog.SetWithoutSource("run_path", testDir)
```
Comment (TCheruy) [inline pkg/collector/corechecks/snmp/snmp_test.go:60]: Can we add a comment to explain why this is needed? just to be sure it's clear for future readers

**Review by jmw51798 — APPROVED** [review]

**Review by git-thuerk-done — APPROVED** [review]
good for docs

File: pkg/collector/corechecks/snmp/snmp_test.go (hunk 1 above)
Comment (jedupau) [reply pkg/collector/corechecks/snmp/snmp_test.go:61]: done https://github.com/DataDog/datadog-agent/pull/25887/commits/8646d253270d88608e89a5918fbd552777a32a16

**Pull request merged**